lib/pure/osproc.nim

```diff
@@ -158,6 +158,15 @@ proc processID*(p: Process): int {.rtl, extern: "nosp$1".} =
   ## returns `p`'s process ID.
   return p.id
 
+proc processID*(): int =
+  ## return current process ID
+  when defined(windows):
+    proc GetCurrentProcessId(): int32 {.stdcall, dynlib: "kernel32",
```

Comment:
int32?
DWORD is unsigned int in Windows headers.

Reply:
fixed, thanks! btw, you can use the new github feature "insert a suggestion" so PR author can just click "accept suggestion"